### PR TITLE
Add japanese translation files

### PR DESCRIPTION
**Pull request by mattn**
* Add japanese translation files
* Add fallback to match Accept-Language to UNIX locale.

Browser send Accept-Language `ja` not `ja_JP`.

### CODE REVIEW

File: notebook/i18n/nbjs.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
 	"domain": "nbjs",
 	"supported_languages": [
 		"fr-FR",
+		"ja",
```
Comment (mattn) [inline notebook/i18n/nbjs.json]: Will fix this soon.

File: notebook/i18n/ja_JP/LC_MESSAGES/nbui.po [hunk 2]
```diff
@@ -0,0 +1,737 @@
...
+msgid "Last Modified"
+msgstr "最終変更時刻"
+
+#: notebook/templates/tree.html:120
+msgid "Name"
+msgstr "名前"
+
+#: notebook/templates/tree.html:130
+msgid "Currently running Jupyter processes"
+msgstr "現在実行風の Jupyter プロセス一覧"
```
Comment (crazymaster) [inline notebook/i18n/ja_JP/LC_MESSAGES/nbui.po]:
```suggestion
msgstr "現在実行中の Jupyter プロセス一覧"
```

Comment (mattn) [reply notebook/i18n/ja_JP/LC_MESSAGES/nbui.po]: Thank you

**Comment by mattn** [comment]
Thanks a lot!

File: notebook/i18n/nbjs.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
 	"domain": "nbjs",
 	"supported_languages": [
 		"fr-FR",
+		"ja_JP",
```
Comment (takluyver) [inline notebook/i18n/nbjs.json]: What browser are you using? This list should probably be consistent in using `-` or `_`, and if different browsers do different things, we may need to work around that.

Comment (mattn) [reply notebook/i18n/nbjs.json]: Google Chrome. 

image

The name of UNIX locale on japanese/japan is `ja_JP`. But the browser send `ja`.

**Pull request merged**